test/make-swappable-proposal.test.ts

```diff
@@ -38,19 +44,57 @@ describe("make swappable proposal", function () {
 
   beforeEach(async function () {
     cowToken = await waffle.deployMockContract(deployer, IERC20.abi);
+    multiTokenMediator = await OmniBridgeTransferSimulator.deploy();
 
     settings = {
       cowToken: cowToken.address,
       virtualCowToken: "0x" + "42".repeat(20),
       atomsToTransfer: "31337",
       multisend: gnosisSafeManager.multisend.address,
+      multiTokenMediator: multiTokenMediator.address,
+      bridged: {
+        virtualCowToken: "0x" + "21".repeat(20),
+        atomsToTransfer: "1337",
+      },
     };
   });
 
+  // Returns an object that assigns to each makeSwappable proposal (flattened)
+  // step the mocks it needs for its execution.
+  function prepareMocks(cowToken: MockContract) {
```

Comment:
Nit: why do you decleare it as a function instead of just a const variable

Reply:
It's definitely unneeded, I removed it! Unfortunately when adding a new test I needed to change one of the parameters, so the function came back with a different parameter.